components/eamxx/src/physics/rrtmgp/eamxx_rrtmgp_process_interface.cpp

```diff
@@ -120,18 +120,32 @@ void RRTMGPRadiation::set_grids(const std::shared_ptr<const GridsManager> grids_
     add_field<Required>("aero_tau_lw", scalar3d_lwband_layout, nondim, grid_name, ps);
   }
 
+  // Whether we do extra clean/clear sky calculations
+  m_extra_clnclrsky_diag = m_params.get<bool>("extra_clnclrsky_diag", false);
```

Comment:
Don't change it, but FYI, if the type can be inferred from the default, you don't need the template. In practice, the _only_ case where you need to specify the type is when you pass a C-string as default, since we store strings as C++ strings. So this
```C++
params.get("my_string","my_value");
```
will fail at compile time (there's some overload that is not specified for C-strings), and you need to instead do
```C++
params.get<std::string>("my_string","my_value");
```
For virtually any other type we use, the provided default value is enough.